cpp/CMakeLists.txt

```diff
@@ -261,9 +261,9 @@ link_directories(
     "${CMAKE_CUDA_IMPLICIT_LINK_DIRECTORIES}")
 
 add_library(cugraph SHARED
-	src/db/db_object.cu
-	src/db/db_parser_integration_test.cu
-	src/db/db_operators.cu
+    src/db/db_object.cu
```

Comment:
why?

Reply:
This was just a formatting change... distracts me when things are indented inconsistently.

Reply:
Oh ok thanks for catching that